Doc/library/os.path.rst

```diff
@@ -306,11 +306,12 @@ the :mod:`glob` module.)
 
 .. function:: normcase(path)
 
-   Normalize the case of a pathname.  On Unix and Mac OS X, this returns the
-   path unchanged; on case-insensitive filesystems, it converts the path to
-   lowercase.  On Windows, it also converts forward slashes to backward slashes.
-   Raise a TypeError if the type of *path* is not ``str`` or ``bytes`` (directly
-   or indirectly through the :class:`os.PathLike` interface).
+   Normalize the case of a pathname.  On Windows and Macintosh, convert all
+   characters in the pathname to lowercase, and on Windows also convert forward
```

Comment:
This seems incorrect. On my Mac it does not lowercase.

Reply:
Python used to work on Mac 9 which was very different from windows or unix, before they made the unix-y Mac OS from a BSD.  In this diff, using «Macintosh» is a bit weird to me.

Reply:
@Mariatta  @merwok I followed the terminology you used in [macpath.py](https://github.com/python/cpython/blob/master/Lib/macpath.py#L1).

Reply:
macpath has also been deprecated and no longer maintained, so I would not copy anything from macpath :)

Everywhere else we've been using `Mac OS X`.

Reply:
Exactly, macpath supports old-style Mac 9, not the current one.  Don’t use that module as a reference.

Reply:
OK, I will change it.